src/windows.rs

```diff
@@ -28,3 +29,24 @@ pub fn kernel_version() -> Option<String> {
         }
     }
 }
+
+/// Get the build number from Windows.
+///
+/// **Note**: On Windows 8 and later this will report the Windows 8 build number 9200,
```

Comment:
Oh, is that so? I somehow thought at least this would always be the _correct_ build number. TIL.

Reply:
Oh oops I thought I had updated this; so the truth is I'm not sure because I don't have multiple Windows pcs to test this on, but it is the behavior I observed. What I do know is that much like the other parts, it's busted without the extra build stuff. Let me update this real quick